packages/langium/src/generator/node-processor.ts

```diff
@@ -35,6 +36,10 @@ class Context {
         return this.lines.map(e => e.join('')).join('');
```

Comment:
That why I prefer to have a `get` or `compute` in names to make it clear there are things happening. Where `this.content.length` looks like a simple field access.